src/FxCop/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -126,4 +126,46 @@
   <data name="DoNotCatchCorruptedStateExceptionsMessage" xml:space="preserve">
     <value>Modify {0} to catch and handle a more specific set of exception type(s) than {1} or rethrow the exception. </value>
   </data>
+  <data name="DoNotUseDES" xml:space="preserve">
+    <value>Do not use insecure cryptographic algorithm DES.</value>
+  </data>
```

Comment:
@genlu  We are doing a complete overhaul of all strings. See document below. We are still working on it. Can I separate the string changes into later pull requests?


https://microsoft.sharepoint.com/teams/CESecEngineering/SecDevTools/Shared%20Documents/Specs,%20Functional%20and%20Design/Roslyn/Roslyn_UserFacingStrings.xlsx?web=1